lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -80,6 +80,12 @@ sub _save_vars {
     close($fd);
 }
 
+sub indent_log_raw_output {
+    my ($text) = @_;
+    $text =~ s/(?<!\A)^/ /gm;
```

Comment:
It would be possible to use the `/r` flag on the substitution, then the return value will be the substituted text. Combined with a signature like @Martchus suggested, it will be a very short one liner:
```
sub indent_log_raw_output($text) { $text =~ s/(?<!\A)^/ /gmr }
```

Reply:
Thanks, done